{"uid":"377fe08be7e70e8a","name":"Refresh token with correct refresh_token","fullName":"tests.test_authentication.test_refresh.TestRefresh#test_refresh","historyId":"9e5d8b034dd8d3eae30196afa6464513","time":{"start":1756231079872,"stop":1756231079872,"duration":0},"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 1 validation error for LoginResponseSchema\ntoken\n  Field required [type=missing, input_value={'detail': 'Invalid credentials'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.11/v/missing","statusTrace":"public_authentication_client = <clients.authentication.public_authentication_client.PublicAuthenticationClient object at 0x7fcb536c3230>\n\n    @pytest.fixture\n    def login_user(public_authentication_client:PublicAuthenticationClient):\n        request=LoginRequestSchema()\n>       response=public_authentication_client.login(request)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nfixtures/authentication.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <clients.authentication.public_authentication_client.PublicAuthenticationClient object at 0x7fcb536c3230>\nrequest = LoginRequestSchema(email='user@example.com', password='string')\n\n    def login(self, request: LoginRequestSchema) -> LoginResponseSchema:\n        response = self.login_api(request)\n>       return LoginResponseSchema.model_validate_json(response.text)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       pydantic_core._pydantic_core.ValidationError: 1 validation error for LoginResponseSchema\nE       token\nE         Field required [type=missing, input_value={'detail': 'Invalid credentials'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.11/v/missing\n\nclients/authentication/public_authentication_client.py:19: ValidationError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1756231077356,"stop":1756231077412,"duration":56},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"save_allure_environment_file","time":{"start":1756231077412,"stop":1756231077412,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"public_authentication_client","time":{"start":1756231079872,"stop":1756231079895,"duration":23},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"login_user","time":{"start":1756231079895,"stop":1756231080165,"duration":270},"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 1 validation error for LoginResponseSchema\ntoken\n  Field required [type=missing, input_value={'detail': 'Invalid credentials'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.11/v/missing\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ~~~~~~~~~~~~~~~~~^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/_pytest/fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/api_autotests/api_autotests/fixtures/authentication.py\", line 26, in login_user\n    response=public_authentication_client.login(request)\n  File \"/home/runner/work/api_autotests/api_autotests/clients/authentication/public_authentication_client.py\", line 19, in login\n    return LoginResponseSchema.model_validate_json(response.text)\n           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/pydantic/main.py\", line 746, in model_validate_json\n    return cls.__pydantic_validator__.validate_json(\n           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^\n        json_data, strict=strict, context=context, by_alias=by_alias, by_name=by_name\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n","steps":[{"name":"Authenticate user","time":{"start":1756231079895,"stop":1756231080165,"duration":270},"status":"passed","steps":[{"name":"Make POST request to '/api/v1/authentication/login'","time":{"start":1756231079896,"stop":1756231080165,"duration":269},"status":"passed","steps":[],"attachments":[{"uid":"38a8c51f2322b7fe","name":"cURL command","source":"38a8c51f2322b7fe.txt","type":"text/plain","size":360}],"parameters":[{"name":"url","value":"'/api/v1/authentication/login'"},{"name":"json","value":"{'email': 'user@example.com', 'password': 'string'}"},{"name":"data","value":"None"},{"name":"files","value":"None"}],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"request","value":"LoginRequestSchema(email='user@example.com', password='string')"}],"stepsCount":1,"shouldDisplayMessage":false,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"stepsCount":2,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 1 validation error for LoginResponseSchema\ntoken\n  Field required [type=missing, input_value={'detail': 'Invalid credentials'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.11/v/missing","statusTrace":"public_authentication_client = <clients.authentication.public_authentication_client.PublicAuthenticationClient object at 0x7fcb536c3230>\n\n    @pytest.fixture\n    def login_user(public_authentication_client:PublicAuthenticationClient):\n        request=LoginRequestSchema()\n>       response=public_authentication_client.login(request)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nfixtures/authentication.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <clients.authentication.public_authentication_client.PublicAuthenticationClient object at 0x7fcb536c3230>\nrequest = LoginRequestSchema(email='user@example.com', password='string')\n\n    def login(self, request: LoginRequestSchema) -> LoginResponseSchema:\n        response = self.login_api(request)\n>       return LoginResponseSchema.model_validate_json(response.text)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       pydantic_core._pydantic_core.ValidationError: 1 validation error for LoginResponseSchema\nE       token\nE         Field required [type=missing, input_value={'detail': 'Invalid credentials'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.11/v/missing\n\nclients/authentication/public_authentication_client.py:19: ValidationError","steps":[],"attachments":[{"uid":"d0056f7fb1e4a4bc","name":"log","source":"d0056f7fb1e4a4bc.txt","type":"text/plain","size":233}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"save_allure_environment_file::0","time":{"start":1756231108897,"stop":1756231108897,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"LMS service"},{"name":"suite","value":"Authentication"},{"name":"feature","value":"Authentication"},{"name":"severity","value":"critical"},{"name":"subSuite","value":"Refresh"},{"name":"epic","value":"LMS service"},{"name":"story","value":"Refresh"},{"name":"tag","value":"refresh_token"},{"name":"tag","value":"regression"},{"name":"tag","value":"authentication"},{"name":"host","value":"pkrvmqc4gcfdwos"},{"name":"thread","value":"2452-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_authentication.test_refresh"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["refresh_token","regression","authentication"]},"source":"377fe08be7e70e8a.json","parameterValues":[]}